book.toml

```diff
@@ -15,6 +15,10 @@ build-dir = "out"
 [preprocessor.mermaid]
 command = "mdbook-mermaid"
 
+[preprocessor.toc]
```

Comment:
Used to generate in-page TOCs, like the one in `Consensus and Storage > Consensus`

Reply:
Yeah why not. Can you also mention it on the top-level `README.md` so non-nix users also get it when they copy&paste the instructions? Thanks